Bug: Z459 should accept Ellipsis

# Bug report

<!--
Hi, thanks for submitting a bug. We appreciate that.

But, we will need some information about what's wrong to help you.
-->

## What's wrong

False positive for `./returns/maybe.pyi:79:5: Z459 Found complex default value`
It should accept `ast.Ellipsis` as a default value, since it is a standard for `.pyi` files.

## How is that should be

```python
@final
class _Nothing(Maybe[Any]):
    _inner_value: None

    def __init__(self, inner_value: None = ...) -> None:
        ...
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -17,6 +17,7 @@ We used to have incremental versioning before `0.1.0`.
 
 - Fixes problem with missing _allowed_left_nodes
 - Fixes problem false positive for Z121 when using `_` for unused var names
+- Fixes problem with false positive for Z459 when a default value is an ellipsis.
 
 ## 0.8.1
 

--- a/tests/test_visitors/test_ast/test_functions/test_complex_default_values.py
+++ b/tests/test_visitors/test_ast/test_functions/test_complex_default_values.py
@@ -9,7 +9,6 @@ from wemake_python_styleguide.visitors.ast.functions import (
     FunctionDefinitionVisitor,
 )
 
-
 function_with_ellipse_default = """
 def __init__(self, inner_value: None = ...) -> None:
     ...

--- a/tests/test_visitors/test_ast/test_functions/test_complex_default_values.py
+++ b/tests/test_visitors/test_ast/test_functions/test_complex_default_values.py
@@ -9,6 +9,13 @@ from wemake_python_styleguide.visitors.ast.functions import (
     FunctionDefinitionVisitor,
 )
 
+
+function_with_ellipse_default = """
+def __init__(self, inner_value: None = ...) -> None:
+    ...
+"""
+
+
 function_with_defaults = """
 def __init__(
     self,
@@ -60,6 +67,22 @@ def test_correct_function_defaults(
 ):
     """Testing that correct function defaults passes validation."""
     tree = parse_ast_tree(mode(function_with_defaults.format(good_default)))
+
+    visitor = FunctionDefinitionVisitor(default_options, tree=tree)
+    visitor.run()
+
+    assert_errors(visitor, [])
+
+
+def test_ellipse_correct_default(
+    assert_errors,
+    assert_error_text,
+    parse_ast_tree,
+    default_options,
+    mode,
+):
+    """Testing that ellipse is accepted as a correct default value."""
+    tree = parse_ast_tree(mode(function_with_ellipse_default))
 
     visitor = FunctionDefinitionVisitor(default_options, tree=tree)
     visitor.run()

--- a/wemake_python_styleguide/violations/best_practices.py
+++ b/wemake_python_styleguide/violations/best_practices.py
@@ -1623,8 +1623,8 @@ class ComplexDefaultValuesViolation(ASTViolation):
     Forbids to use complex defaults.
 
     Anything that is not a ``ast.Name``, ``ast.Attribute``, ``ast.Str``,
-    ``ast.NameConstant``, ``ast.Tuple``, ``ast.Bytes``, ``ast.Num``  or ``ast.Ellipsis``
-    should be moved out from defaults.
+    ``ast.NameConstant``, ``ast.Tuple``, ``ast.Bytes``, ``ast.Num``
+    or ``ast.Ellipsis`` should be moved out from defaults.
 
     Reasoning:
         It can be tricky. Nothing stops you from making database calls or http

--- a/wemake_python_styleguide/violations/best_practices.py
+++ b/wemake_python_styleguide/violations/best_practices.py
@@ -1623,8 +1623,8 @@ class ComplexDefaultValuesViolation(ASTViolation):
     Forbids to use complex defaults.
 
     Anything that is not a ``ast.Name``, ``ast.Attribute``, ``ast.Str``,
-    ``ast.NameConstant``, ``ast.Tuple``, ``ast.Bytes`` or ``ast.Num`` should
-    be moved out from defaults.
+    ``ast.NameConstant``, ``ast.Tuple``, ``ast.Bytes``, ``ast.Num``  or ``ast.Ellipsis``
+    should be moved out from defaults.
 
     Reasoning:
         It can be tricky. Nothing stops you from making database calls or http

--- a/wemake_python_styleguide/visitors/ast/functions.py
+++ b/wemake_python_styleguide/visitors/ast/functions.py
@@ -107,7 +107,7 @@ class FunctionDefinitionVisitor(BaseNodeVisitor):
         ast.Tuple,
         ast.Bytes,
         ast.Num,
-        ast.Ellipsis
+        ast.Ellipsis,
     )
 
     def _check_used_variables(

--- a/wemake_python_styleguide/visitors/ast/functions.py
+++ b/wemake_python_styleguide/visitors/ast/functions.py
@@ -107,6 +107,7 @@ class FunctionDefinitionVisitor(BaseNodeVisitor):
         ast.Tuple,
         ast.Bytes,
         ast.Num,
+        ast.Ellipsis
     )
 
     def _check_used_variables(

